org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/services/XMLSymbolsProvider.java

```diff
@@ -362,29 +363,51 @@ private boolean processSymbolsParticipants(DOMDocument xmlDocument, SymbolInform
 		// Process replace symbol providers participants
 		Collection<ISymbolsProviderParticipant> replaceParticipants = resultParticipant.getReplaceParticipants();
 		if (!replaceParticipants.isEmpty()) {
+			boolean successfulReplace = false;
 			if (replaceParticipants.size() > 1) {
 				LOGGER.log(Level.WARNING, "There are '" + replaceParticipants.size() + "' replace participants");
 			}
 			for (ISymbolsProviderParticipant replaceParticipant : replaceParticipants) {
-				if (symbolInformations != null) {
-					replaceParticipant.findSymbolInformations(xmlDocument, symbolInformations, filter, cancelChecker);
-				}
-				if (documentSymbols != null) {
-					replaceParticipant.findDocumentSymbols(xmlDocument, documentSymbols, filter, cancelChecker);
+				try {
+					if (symbolInformations != null) {
+						replaceParticipant.findSymbolInformations(xmlDocument, symbolInformations, filter, cancelChecker);
+					}
+					if (documentSymbols != null) {
+						replaceParticipant.findDocumentSymbols(xmlDocument, documentSymbols, filter, cancelChecker);
+					}
+					successfulReplace = true;
+				} catch (CancellationException e) {
+					throw e;
+				} catch (Exception e) {
+					LOGGER.log(Level.SEVERE,
+							"Error while processing symbols for '" + replaceParticipant.getClass().getName() + "': " + e.getMessage());

```

Comment:
Pass exception as third parameter, end with `.` instead of `:`

Reply:
yes please do that